iree/runtime/BUILD

```
@@ -4,12 +4,24 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+# Generic (bare-metal) system doesn't have the file system support.
+iree_cmake_extra_content(
+    content = """
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Generic")
```

nit: style (but see my other comment, we should remove this check)
```suggestion
if(${CMAKE_SYSTEM_NAME} STREQUAL "Generic")
```

---

Style fixed. Add a todo to address the issue.